server/views/pages/overview/partials/_actionsCard.njk

```diff
@@ -0,0 +1,37 @@
+
+<div class="govuk-summary-card actions-card govuk-!-display-none-print" data-qa="actions-card">
+  <div class="govuk-summary-card__title-wrapper govuk-!-padding-bottom-2">
+    <h2 class="govuk-summary-card__title govuk-!-font-size-24">Actions</h2>
+  </div>
+
+  {# Review based actions #}
+  {% set showReviewActions = not induction.problemRetrievingData and induction.isPostInduction and not actionPlanReview.problemRetrievingData and reviewJourneyEnabledForPrison%}
+  {% if showReviewActions %}
+    <div class="govuk-summary-card__content govuk-!-padding-bottom-0 govuk-!-padding-top-2">
+      <h3 class="govuk-heading-s govuk-!-margin-bottom-0">Reviews</h3>
+      <span class="govuk-tag govuk-!-margin-top-2 govuk-!-margin-bottom-2 govuk-tag--grey">Review due by 29 Sep 2024</span>
```

Comment:
I think all we need to do is change this `span` to put the correct tag here;

and then add some `li`'s (wrapped in `if hasEditAuthority`) within the `ul` .... I _think_ that should be all we need to do 🤞